src/mlagility/cli/report.py

```diff
@@ -1,521 +1,204 @@
 import os
 import csv
-import dataclasses
-from typing import List
-import yaml
+import json
+from datetime import datetime
+from pathlib import Path
+from dataclasses import dataclass
 import groqflow.common.printing as printing
 import groqflow.common.build as build
 import groqflow.common.cache as cache
-from groqflow import groqmodel
+from groqflow.groqmodel import groqmodel
+from mlagility.common import labels
 
 
-def _successCleanup(item):
-    return item if item is not None else False
+def _update_numeric_attribute(new_val, current_val, default="-"):
+    """
+    Updates a numeric attribute if needed
+    """
+    if current_val == default:
+        return new_val if new_val is not None else default
+    else:
+        return current_val
 
 
-def _numericCleanup(item):
-    return item if item is not None else "-"
+def get_estimated_groq_latency(model_folder, cache_folder):
```

Comment:
```suggestion
def _get_estimated_groq_latency(model_folder, cache_folder):
```

if it wont be imported anywhere

Reply:
Accepted!